src/routers/Gateway.sol

```diff
@@ -79,14 +125,52 @@ contract ConnectorGateway {
         emit Deny(user);
     }
 
+    function pause() external onlyPauseAdmin {
```

Comment:
I don't think we need to store `pauseAdmin` and `delayedAdmin` here. We can just add the `auth `modifier to `pause()`, `unpause()`, `scheduleRely48hr()`, and `cancelSchedule()`